.ci-scripts/test-library

```diff
@@ -8,4 +8,4 @@ rm -f /tmp/create-vanilla-SCDB.sh
 wget -q https://raw.githubusercontent.com/quattor/scdb/master/utils/scdb/create-vanilla-SCDB.sh -O /tmp/create-vanilla-SCDB.sh
 chmod u+x /tmp/create-vanilla-SCDB.sh
 
-/tmp/create-vanilla-SCDB.sh -F --debug --continuous-integration HEAD
+/tmp/create-vanilla-SCDB.sh -F --continuous-integration HEAD
```

Comment:
Was removing `--debug` here intentional?

Reply:
Yes. I don't think it is useful. At least it is not used in other repositories and makes the output very verbose, despite it is unlikely to be useful...